src/Integration.Vsix.UnitTests/CFamily/CFamilyReproducerCommandTests.cs

```diff
@@ -94,6 +103,7 @@ public void CheckStatus_NoActiveDocument_NotEnabled()
 
             VerifyDocumentLocatorCalled();
             VerifyLanguageRecognizerNotCalled();
+            VerifyAnalysisNotRequested();
```

Comment:
From these tests, I understand that Execute evaluated when "OleStatus" is accessed, is that correct?
I think it will be worth creating a separate test, to explicitly test for the analysis request: `CheckStatus_NoActiveDocument_AnalyzerNotCalled` and etc

Reply:
As discussed offline, I've added the check to the other `CheckStatus` test.